seed/lib/mappings/data/pm-mapping.json

```diff
@@ -130,7 +130,7 @@
     "to_field": "gross_floor_area",
     "to_table_name": "PropertyState",
     "from_field": "propGrossFloorArea",
-    "units": "",
+    "units": "ft**2",
```

Comment:
I like that we can take what were once just cosmetic units and actually make them useful!